lib/config/options/index.ts

```diff
@@ -1218,6 +1218,7 @@ const options: RenovateOptions[] = [
     description:
```

Comment:
Maybe this should be a breaking change PR, as the change may close/open a lot of PRs?